css/styles.css

```diff
@@ -1,83 +1,103 @@
 body {
-    background-color: white;
-    font-family: sans-serif;
-    padding: 20px;
-    margin: 0%;
-    padding: 0%;
-    overflow-y: scroll;
+  background-color: white;
+  font-family: sans-serif;
+  padding: 20px;
```

Comment:
It is a good practice to establish at least families of letters, this helps the browser to have options to choose from.  I suggest adding one more font family.